zebra-state/src/config.rs

```diff
@@ -45,7 +45,7 @@ pub struct Config {
     /// | Linux   | `$XDG_CACHE_HOME/zebra` or `$HOME/.cache/zebra` | `/home/alice/.cache/zebra`           |
     /// | macOS   | `$HOME/Library/Caches/zebra`                    | `/Users/Alice/Library/Caches/zebra`  |
     /// | Windows | `{FOLDERID_LocalAppData}\zebra`                 | `C:\Users\Alice\AppData\Local\zebra` |
-    /// | Other   | `std::env::current_dir()/cache/zebra`           | `/cache/zebra`                       |
+    /// | Other   | `std::env::current_dir()/.cache/zebra`          | `/.cache/zebra`                      |
```

Comment:
Restored in https://github.com/ZcashFoundation/zebra/pull/8143/commits/7a24e455677872e1a169ccf13c8b59c8b8a2e9a9.

It may be `.cache` for some OSes, but we don't officially support those and `.cache` is already listed as a possibility for Linux.